files/en-us/mdn/contribute/localize/index.html

```diff
@@ -60,11 +60,10 @@ <h3 id="Russian_ru">Russian (ru)</h3>
  <li>Current contributors: <a href="https://github.com/armanpwnz">armanpwnz</a>, <a href="https://github.com/captainspring">captainspring</a>, <a href="https://github.com/mpstv">mpstv</a>, <a href="https://github.com/myshov">myshov</a>, <a href="https://github.com/Saionaro">Saionaro</a>, <a href="https://github.com/sashasushko">sashasushko</a>, <a href="https://github.com/lex111">lex111</a></li>
 </ul>
 
-<h2 id="Other_MDN_localization">Other MDN localization</h2>
-
-<p>For the moment, the new MDN platform UI will only be in English. This is a problem that we are going to get to later on.</p>
+<div class="notecard note">
+  <p><strong>Note</strong>: If you want to discuss unfreezing a currently frozen locale, the <a href="https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale">guidelines on what is required can be found here</a>.</p>
```

Comment:
The colon must be inside the tag:
```suggestion
  <p><strong>Note:</strong> If you want to discuss unfreezing a currently frozen locale, the <a href="https://github.com/mdn/translated-content/blob/main/PEERS_GUIDELINES.md#activating-a-locale">guidelines on what is required can be found here</a>.</p>
```